templates/docs/examples/patterns/buttons/small.html

```diff
@@ -0,0 +1,8 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Buttons / Small{% endblock %}
+
+{% block standalone_css %}patterns_buttons{% endblock %}
+
+{% block content %}
+<small>This is small text <button class="p-button--neutral is-small">This is a small button</button><button class="p-button--neutral is-small is-dense">This is a small, dense button</button></small>
```

Comment:
Could we wrap it in paragraph as well to make sure it's properly placed on baseline grid?

```suggestion
<p><small>This is small text <button class="p-button--neutral is-small">This is a small button</button><button class="p-button--neutral is-small is-dense">This is a small, dense button</button></small></p>
```